extensions/amp-analytics/amp-analytics.md

```diff
@@ -425,7 +425,38 @@ This event is also emitted by the document itself and can be configured as:
 "triggers": {
   "renderStart": {
     "on": "render-start",
-    "request": "render-start"
+    "request": "request"
+  }
+}
+```
+
+#### Initial load trigger (`"on": "ini-load"`)
+This event is triggered when initial contents of an element or a document have been loaded.
```

Comment:
i guess it's not supported for all elements right? shall we list the ones that support this?